pyuvsim/simsetup.py

```diff
@@ -1727,14 +1750,20 @@ def initialize_uvdata_from_params(obs_params, return_beams=None):
 
     uv_obj.ant_1_array, uv_obj.ant_2_array = uv_obj.baseline_to_antnums(uv_obj.baseline_array)
 
+    cat_id = uv_obj._add_phase_center(
+        cat_name=uvparam_dict["cat_name"], cat_type="unprojected"
+    )
+    uv_obj.phase_center_id_array = np.full(uv_obj.Nblts, cat_id, dtype=int)
+
     # add other required metadata to allow select to work without errors
     # these will all be overwritten in uvsim._complete_uvdata, so it's ok to hardcode them here
 
     _set_lsts_on_uvdata(uv_obj)
+    uv_obj._set_app_coords_helper()
```

Comment:
what does this do?

Reply:
This sets the apparent coordinates on the object, which are in the process of becoming required.